app/controllers/api/v1/basic_auth_api/protocol_subscriptions_controller.rb

```diff
@@ -27,7 +27,7 @@ def create
         end
 
         def delegated_protocol_subscriptions
```

Comment:
this is quite the change tbh if you're relying on this api to be stable. We should probably also document this in the future somewhere

Reply:
Yep. Here's the thing: it's only used for SDV currently. That the only reason that I was able to make this change.